### ISSUE TITLE
Heterogeneous storage

### ISSUE THREAD

**Issue by avikivity** [scylladb/scylla#1315]
Store commit logs and smaller memtables on storage-class memory; compact to SSDs, pushing less frequently use objects to spinning disks.

This allows large amounts of storage while keeping costs down.

**Comment by denesb** [comment]
I think this is superseeded by S3 effort, no?

**Comment by regevran** [comment]
> I think this is superseeded by S3 effort, no?

Yes. Tiered storage may be implemented in various ways. S3 is one of them, the one suggested here is another one.
However, I think we should close this issue as spinning disks are not part of the tiering effort.

**Comment by denesb** [comment]
Ok, closing. We decided to use S3 as slower/cheaper storage tier, instead of spinning disks.
